fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java

```diff
@@ -274,6 +273,11 @@ public enum ErrorCode {
      */
     ERR_LOCK_ERROR(5300, new byte[] {'5', '5', 'P', '0', '3'}, "Failed to acquire lock: %s"),
     ERR_BEGIN_TXN_FAILED(5301, new byte[] {'5', '5', 'P', '0', '3'}, "Failed to begin transaction: %s"),
+    ERR_TXN_NOT_EXIST(5302, new byte[] {'2', '5', '0', 'P', '1'}, "Transaction %s does not exist"),
```

Comment:
when will this error occur?

Reply:
This is just a defensive code to prevent transactionMgr from returning null